node/bft/events/src/primary_ping.rs

```diff
@@ -19,6 +19,7 @@ use super::*;
 pub struct PrimaryPing<N: Network> {
     pub version: u32,
     pub block_locators: BlockLocators<N>,
+    pub certificate_id: Field<N>,
```

Comment:
Note: Changes to PrimaryPIng or messages in general will necessitate an update to the `Event` Version.